internal/hrq/hrqreconciler.go

```diff
@@ -186,6 +190,32 @@ func (r *HierarchicalResourceQuotaReconciler) syncUsages(inst *api.HierarchicalR
 	}
 	inst.Status.Used = utils.FilterUnlimited(usage, inst.Spec.Hard)
 
+	// Update status.request and status.limit to show HRQ status by using kubectl get
+	resources := make([]v1.ResourceName, 0, len(inst.Status.Hard))
+	for resource := range inst.Status.Hard {
+		resources = append(resources, resource)
+	}
+	sort.Sort(sortableResourceNames(resources))
+
+	requestColumn := bytes.NewBuffer([]byte{})
+	limitColumn := bytes.NewBuffer([]byte{})
+	for i := range resources {
+		w := requestColumn
+		resource := resources[i]
+		usedQuantity := inst.Status.Used[resource]
+		hardQuantity := inst.Status.Hard[resource]
+
+		// use limitColumn writer if a resource name prefixed with "limits" is found
+		if pieces := strings.Split(resource.String(), "."); len(pieces) > 1 && pieces[0] == "limits" {
+			w = limitColumn
+		}
+
+		fmt.Fprintf(w, "%s: %s/%s, ", resource, usedQuantity.String(), hardQuantity.String())
```

Comment:
I prefer the following, but what do you think? I am concerned that each variable is defined in a location far from where it is used.

```go
		resource := resources[i]

		var w *bytes.Buffer
		if strings.HasPrefix(resource.String(), "requests.") {
			w = requestColumn
		} else {
			w = limitColumn
		}

		usedQuantity := inst.Status.Used[resource]
		hardQuantity := inst.Status.Hard[resource]
		fmt.Fprintf(w, "%s: %s/%s, ", resource, usedQuantity.String(), hardQuantity.String())
```